sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -970,6 +970,8 @@ SIREPO.app.factory('plotting', function(appState, frameCache, panelState, utilit
             };
         },
 
+        randomId: () => Math.floor(Math.random() * Number.MAX_SAFE_INTEGER),
+
```

Comment:
Might as well use `SIREPO.UTILS.randomString()`

Reply:
I added SIREPO.UTILS.randomId() which uses randomString(). The HTML id spec recommends starting ids with a letter.

Reply:
Cool